packages/react-native-drawer-layout/src/views/modern/Drawer.tsx

```diff
@@ -310,7 +310,7 @@ export function Drawer({
         : minmax(translationX.value - touchDistance, 0, drawerWidth);
 
     return translateX;
-  });
+  }, [touchStartX, drawerWidth, gestureState, drawerPosition, translationX]);
```

Comment:
With `react-native-reanimated/plugin` auto-generated dependencies would be:
`[drawerType, gestureState, GestureState, minmax, drawerPosition, touchStartX, drawerWidth, layout, translationX]`. Unless there's good reason not to include some of them here I'd rather keep them all.

Reply:
Thanks! I am updating this, however GestureState is an enum, I think we shouldn't put enums as dependencies.
![Screenshot 2023-08-04 at 17 07 54](https://github.com/react-navigation/react-navigation/assets/1704434/5b185285-c443-4f35-a6ce-14cb82d9cf1b)

Reply:
Why shouldn't we? Had it been transpiled it would've been captured into dependencies.

Reply:
Because it is a constant and it will never change.

Also, I am receiving the typescript error displayed above.

Reply:
Well, in the future it could be mutable (although I strongly doubt it) and enums are transpiled into JS objects, but I guess it's fair enough to not include it here.